nix/update-wlroots.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env -S nix shell nixpkgs#gawk nixpkgs#git nixpkgs#gnused nixpkgs#jq nixpkgs#ripgrep -c bash
+
+# get wlroots revision from submodule
+SUB_REV=$(git submodule status | rg wlroots | awk '{ print substr($1,1)}')
```

Comment:
Is it supposed to be `($1,2)`?

Reply:
Yeah, for some reason it truncated the first character of the rev on my machine. I'll fix it.